launchToolbar/js/launchToolbar_meta.user.js

```diff
@@ -3,14 +3,14 @@
 // @namespace www.cobaltgroup.com
 // @include https://cdk.my.salesforce.com/a1E*
 // @include https://cdk--c.na57.visual.force.com/*
-// @version 4.8.2
+// @version 4.8.3
 // @run-at document-end
-// @downloadURL https://raw.githubusercontent.com/cirept/salesforceToolbar/master/launchToolbar/js/launchToolbar_meta.user.js
+// @downloadURL https://github.com/cirept/salesforceToolbar/raw/master/launchToolbar/js/launchToolbar_meta.user.js
 // @require https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js
 // @resource fontAwe https://use.fontawesome.com/3953f47d82.js
 // @resource googFont https://fonts.googleapis.com/css?family=Raleway:600,900
 // @resource jqueryUI https://ajax.googleapis.com/ajax/libs/jqueryui/1.12.1/themes/smoothness/jquery-ui.css
-// @require https://raw.githubusercontent.com/cirept/salesforceToolbar/v4.8.2/launchToolbar\js\launchToolbar.min.js
+// @require https://github.com/cirept/salesforceToolbar/raw/master/launchToolbar/js/launchToolbar.min.js
```

Comment:
This links to the .min.js file.  I don't believe you have minified your file, so please link it to the main launchToolbar.js file.